examples/pyfluent-workflow/main.py

```diff
@@ -0,0 +1,234 @@
+
+fluent_session.setup.boundary_conditions.pressure_outlet[
+    "outlet"
+].turbulence.backflow_turbulent_viscosity_ratio = 4
+
+# ## Initialize flow field
+
+fluent_session.solution.initialization.hybrid_initialize()
+
+# ## Solve for 150 iterations
+# Setting iteration count to 150 to solve the model.
+
+dir()
+fluent_session.solution.run_calculation.iter_count = 100
+
+
+# ## Update Solution using Workbench Journal Commands
+
+script_string = """
+solutionComponent1 = system1.GetComponent(Name="Solution")
+system1 = GetSystem(Name="FLU")
+solutionComponent1 = system1.GetComponent(Name="Solution")
+solutionComponent1.Update(AllDependencies=True)
+"""
+
+wb.run_script_string(script_string)
+
+
+# ## Postprocessing
+# Create and display velocity vectors on the ``symmetry-xyplane`` plane.
+
+# ## Configure graphics picture export
+# Since Fluent is being run without the GUI, you must to export plots as picture files. Edit the picture settings to use a custom resolution so that the images are large enough.
+
+graphics = fluent_session.results.graphics
+if graphics.picture.use_window_resolution.is_active():
+    graphics.picture.use_window_resolution = False
+graphics.picture.x_resolution = 1920
+graphics.picture.y_resolution = 1440
+
+# ## Create velocity vectors
+# Create and display velocity vectors on the ``symmetry-xyplane`` plane. Then, export the image for inspection.
+
+graphics = fluent_session.results.graphics
+
+graphics.vector["velocity_vector_symmetry"] = {}
+velocity_symmetry = fluent_session.results.graphics.vector["velocity_vector_symmetry"]
+velocity_symmetry.print_state()
+velocity_symmetry.field = "velocity-magnitude"
+velocity_symmetry.surfaces_list = [
+    "symmetry-xyplane",
+]
+velocity_symmetry.scale.scale_f = 4
+velocity_symmetry.style = "arrow"
+velocity_symmetry.display()
+
+graphics.views.restore_view(view_name="front")
+graphics.views.auto_scale()
+graphics.picture.save_picture(file_name="velocity_vector_symmetry.png")
+
+# ## Compute mass flow rate
+# Compute the mass flow rate.
+
+# +
+fluent_session.solution.report_definitions.flux["mass_flow_rate"] = {}
+
+mass_flow_rate = fluent_session.solution.report_definitions.flux["mass_flow_rate"]
+mass_flow_rate.boundaries = [
+    "cold-inlet",
+    "hot-inlet",
+    "outlet",
+]
+mass_flow_rate.print_state()
+fluent_session.solution.report_definitions.compute(report_defs=["mass_flow_rate"])
+# -
+
+# ## Save project
+
+save_string = """import os
+workdir = GetServerWorkingDirectory()
+path = os.path.join(workdir, "mixing_elbow.wbpj")
+Save(FilePath=path , Overwrite=True)"""  
+wb.run_script_string(save_string)
+
+# ## Archive Project
+
+archive_string ="""import os
+workdir = GetServerWorkingDirectory()
+path = os.path.join(workdir, "mixing_elbow.wbpz")
+Archive(FilePath=path , IncludeExternalImportedFiles=True)"""  
+wb.run_script_string(archive_string)
+
+# ## Download the saved project which has all simulation data and results.
+
+wb.download_file("mixing_elbow.wbpj")
```

Comment:
Do you mean to download the project archive (wbpz file) here instead? @abhishekchitwar